packages/dds/tree/src/index.ts

```diff
@@ -101,14 +102,15 @@ export {
 	// Can not be moved to internalTypes since doing so causes app code to throw errors like:
 	// error TS2742: The inferred type of 'Inventory' cannot be named without a reference to '../node_modules/@fluidframework/tree/lib/internalTypes.js'. This is likely not portable. A type annotation is necessary.
 	type AllowedTypes,
-	type WithType,
 	type TreeObjectNodeUnsafe,
 	type InsertableTreeNodeFromImplicitAllowedTypesUnsafe,
 	type TreeArrayNodeUnsafe,
 	type TreeMapNodeUnsafe,
 	type InsertableObjectFromSchemaRecordUnsafe,
 	type InsertableTreeFieldFromImplicitFieldUnsafe,
 	type FieldSchemaUnsafe,
+	// System types (not in Internal types for various reasons, like doc links or cannot be named errors).
+	getJsonSchema,
```

Comment:
This function isn't a `@system` type. Should probably go somewhere else.

Reply:
Messed up some copy paste there. Fixed.